spec/std/system_error_spec.cr

```diff
@@ -0,0 +1,11 @@
+require "spec"
+
+describe SystemError do
+  describe ".from_os_error" do
+    it "Can build an error from an errno" do
+      errno = Errno.new(11)
+      error = ::IO::Error.from_os_error(message: nil, os_error: errno)
+      error.message.should eq("Resource temporarily unavailable")
```

Comment:
LibC-specific. It will be interesting to see if it passes CI.

Reply:
It doesn't. On Mac it gives `"Resource deadlock avoided"`. I'll change to match against /Resource/ unless someone has a better idea.

Reply:
BTW, it is a bit strange to get email about failed specs, but that it doesn't show up as broken in the github UI. Is that a known issue?

Reply:
Maybe the CI notification is for an older commit?

Reply:
Can't be, I got two failure notifications and no obvious sign in the UI. But perhaps it was there but just not obvious enough :shrug:

Reply:
For the error messages, you can just compare the respective manpages (https://man7.org/linux/man-pages/man3/errno.3.html, https://developer.apple.com/library/archive/documentation/System/Conceptual/ManPages_iPhoneOS/man2/intro.2.html, https://www.freebsd.org/cgi/man.cgi?query=errno&sektion=2&format=html, etc.).

I suppose `ENOENT` should be pretty portable: `No such file or directory`.